### PR TITLE
Update RC verification with release directory handling.

### PR DESCRIPTION
**Pull request by MJJoyce**

### CODE REVIEW

File: verify_gpg_sigs [hunk 1]
```diff
@@ -14,10 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # 
+# ./verify_gpg_sigs [RELEASE_DIRECTORY]
+
+if [ -z "$1" ]; then
+    cd $1
+fi
+
+cd $1
```
Comment (chrismattmann) [inline verify_gpg_sigs]: shouldn't this be removed?

File: verify_md5_checksums [hunk 2]
```diff
@@ -14,6 +14,14 @@
...
 # limitations under the License.
 #
+# Usage:
+#   ./verify_md5_checksums [RELEASE_DIRECTORY]
+
+if [ -z "$1" ]; then
+    cd $1
+fi
+
+cd $1
```
Comment (chrismattmann) [inline verify_md5_checksums]: shouldn't this be removed?

**Comment by MJJoyce** [comment]
Ah +1 @chrismattmann, I was rebasing and cleaning up commits and I failed miserably at dealing with merge conflicts. One moment and I'll clean that up for ya =)

**Comment by chrismattmann** [comment]
@autoack #bewm #merged

**Pull request merged**